src/Assetic/Filter/ScssphpFilter.php

```diff
@@ -99,9 +104,12 @@ public function addImportPath($path)
         $this->importPaths[] = $path;
     }
 
-    public function registerFunction($name, $callable)
+    public function registerFunction($name, $callable, $argumentDeclaration = null)
```

Comment:
you could typehint the new argument as `array`

Reply:
Thanks @stof